docs/runbooks/CDINotReady.md

```diff
@@ -0,0 +1,31 @@
+# CDINotReady
+
+## Meaning
+
+The CDINotReady alert is indicative of a CDI installation being in a degraded state;  
+- Not progressing
+- Not available to use
+
+## Impact
+
+CDI is simply not usable, its components are not ready & stopped progressing towards a ready state and hence require action.
```

Comment:
@akalenyu Please update what it means its not usable, what can't the user/system do if this happens?

Reply:
Done